.github/workflows/ci.yaml

```diff
@@ -16,6 +16,7 @@ jobs:
         ruby-version:
         - '3.0'
         - '3.1'
+        - '3.3'
```

Comment:
@kbrock Even if we jump from 3.1 to 3.3 in a release, we should include 3.2 with 3.3 so if they fail in CI, we can determine if it's a problem unique to 3.3 or also a problem in 3.2

Reply:
Yea, since we will never use 3.2, I'm mixed on this one.
Already got 2 thumbs up, so I'll add it.
But it seems a waste of cycles and unnecessary granularity